provider/aws/formation/rack.json

```diff
@@ -2014,8 +2025,11 @@
         "Description": { "Fn::Join": [ "", [ "{\"Cluster\": \"", { "Ref": "Cluster" }, "\", \"Rack\": \"", { "Ref": "AWS::StackName" }, "\"}" ] ] },
         "Environment": {
           "Variables": {
+            "AS_ENDPOINT": { "Fn::Select": [ 1, { "Fn::Split": [ ":", { "Fn::Select": [ 0, { "Fn::GetAtt": [ "ASEndpoint", "DnsEntries" ] } ] } ] } ] },
```

Comment:
1. The `{ "Fn::GetAtt": [ "ASEndpoint", "DnsEntries" ] }` returns a list of values
2. But we only need the first one: `{ "Fn::Select": [ 0, { "Fn::GetAtt": [ "ASEndpoint", "DnsEntries" ] } ] }`
3. the value returned by DnsEntries contain the hosted zone id then the dns entry, so we need to split: `{ "Fn::Split": [ ":", { "Fn::Select": [ 0, { "Fn::GetAtt": [ "ASEndpoint", "DnsEntries" ] } ] } ] }`
4. Then we only need the DNS entry, so we get only the index 1: `{ "Fn::Select": [ 1, { "Fn::Split": ...`